### ISSUE TITLE
[COGNITION]: Disability Calculator - Can we mention the expected data format in on-page instructions?

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#1045]
## Description

Our quick test logged an improvement item to include our preferred 10% increment instructions in the calculator descriptive paragraph, not just the error message. Screenshot attached below, but might need design consult for good layout.

/cc @smoonpeyes 

## Related Issues
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/1039

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* As a user, I want to know how I should enter data, IE 10, 20, 30... in the Disability Rating field, before the error message appears on screen.

## Environment

* MacOS
* Chrome

## WCAG or Vendor Guidance (optional)

* [Labels or Instructions: Understanding SC 3.3.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-cues.html)

## Screenshots or Trace Logs

![Screen Shot 2019-08-16 at 11 18 26 AM](https://user-images.githubusercontent.com/934879/63182307-e62cfe80-c017-11e9-9808-d96e807dbf65.png)

**Comment by rtwell** [comment]
@jenniferlee-dsva thoughts? I defer to you for this one

**Comment by jenniferlee-dsva** [comment]
Hi @1Copenut - No for numerous reasons - 1/ we already provide that info on the body content of page (the new updated Danielle's version that's pending stakeholder approval at the mo) and 2/ don't want to clutter up an already instruction-heavy tool interface. 

Our hypothesis is that users who have ratings from VA will come here to plug in their numbers, so we *think* users will mainly already know to enter their real VA numbers from their letters. We believe the error will not occur that often, hence providing that detail as a message WHEN the user makes the error. 

The tool is getting analytics tracking, so we should be able to track how frequently ppl are erroring, and  better assess then if this error message really needs to be an upfront instruction. 

Love all your ideas though. Thank you. 

I'm supposed to hear back from SMEs ok/no okay by eod Weds - once the page content is okayed, we can move ahead with adding the calculator to the new page.

**Comment by aeltarhoni** [comment]
Iceboxing until work is warranted post launch.

**Comment by 1Copenut** [comment]
Thanks all. I'm happy to discuss further as warranted. I can appreciate the user model defined above, and know we'll find a solution that checks all the boxes.

**Comment by 1Copenut** [comment]
@aeltarhoni I'd like to revisit this item. The VA 508 office identified the entry field as needing some instruction for how to enter data.

> When I enter in the disability rating field there is not example displayed or read on what format to annotate in the edit field – do I write eighty percent, 80 or 80%?  Now I do get an error message if I do it wrong but the error does not address how I should annotate in the field.

Let's review this tomorrow or Monday, and we can discuss some options to meet this success criterion.

**Comment by brianalloyd** [comment]
Following up on this ticket, @1Copenut or @rtwell any decisions in terms of direction for this element, might have some room in this sprints burn down to action.  Thanks.

**Comment by rtwell** [comment]
ok gang — I consulted with @1Copenut and here is our proposed solution:

![image](https://user-images.githubusercontent.com/12564977/125814333-97e44d8d-278d-4bc5-8607-79a4eda1c57f.png)

1. Add the sentence "Disability ratings are given in 10% increments, from 0 to 100." to the paragraph beneath the tool header
2. On desktop, we display contextually relevant hint text, leveraging we use the [hint text component](https://design.va.gov/experimental-design/hint-text) developed in the Experimental Design System
3. On mobile, we drop the hint text in lieu of bolding "Disability ratings are given in 10% increments, from 0 to 100." in the paragraph above.

**Comment by brianalloyd** [comment]
@rtwell PW team has slotted this update into next sprint.  Thanks for the updated context and visuals in support.

**Comment by Rleahy22** [comment]
@brianalloyd while working on this ticket I noticed that the inputs don't exactly match up with the instructions. It says ratings are **from** 0-100, but both 0 and 100 are actually considered invalid. See the screenshot with validation errors:
<img width="711" alt="Screen Shot 2021-08-17 at 1 59 35 PM" src="https://user-images.githubusercontent.com/3144003/129777277-0bab74ab-fe45-44d8-ab5a-9b4f4cf752e8.png">

I would recommend we either change the text to say **between** 0 and 100, or change the min to 0 and max to 100. My preference is for the latter but I don't know if those values would actually break validation further down the system. Thoughts?

**Comment by rtwell** [comment]
Let's change the text to `between`. thank you!

**Comment by brianalloyd** [comment]
@Rleahy22 as @rtwell indicated above, change text to `between` as suggested.  Thanks for working this effort Ryan, really appreciate it.

**Comment by cmaeng** [comment]
it looks like recent work on this triggered an [accessibility alert](https://github.com/department-of-veterans-affairs/va.gov-team/issues/28978) to go off. it appears that the contrast for the text isn't high enough.

<img width="1575" alt="Screen Shot 2021-08-19 at 12 23 04 PM" src="https://user-images.githubusercontent.com/60702623/130132599-65d8c217-8cf5-44da-b20b-828304f51d0b.png">

**Comment by rtwell** [comment]
ahhhh crap — that is totally an oversight on my part. let's make it the same color as the rest of the type on the page

**Comment by Rleahy22** [comment]
Moving to validate: The hint text color has been updated, and the description now reads "between 0 and 100." This should go out with the 8/20 deployment.

**Comment by brianalloyd** [comment]
@rtwell I validated this on my end and everything looks good.  Would you mind giving it a look ISO validation as well.  Thanks.

**Comment by brianalloyd** [comment]
Looks good on my end, closing in support.  We can reopen if any issues are identified.
